sites/shared/components/workbench/draft/circle/index.js

```diff
@@ -1,20 +1,16 @@
-const Circle = (props) => props.point.attributes.getAsArray('data-circle').map((r,i) => {
-  const circleProps = props.point.attributes.asPropsIfPrefixIs('data-circle-')
-  const extraProps = {}
-  for (const prop in circleProps) {
-    const val = props.point.attributes.getAsArray(`data-circle-${(prop === 'className' ? 'class' : prop)}`)
-    if (val.length >= i) extraProps[prop] = val[i]
-    else extraProps[prop] = val.join(' ')
-  }
+const Circle = (props) =>
+  props.point.attributes.getAsArray('data-circle').map((r, i) => {
+    const circleProps = props.point.attributes.asPropsIfPrefixIs('data-circle-')
+    const extraProps = {}
+    for (const prop in circleProps) {
+      const val = props.point.attributes.getAsArray(
+        `data-circle-${prop === 'className' ? 'class' : prop}`
+      )
+      if (val.length >= i) extraProps[prop] = val[i]
+      else extraProps[prop] = val.join(' ')
+    }
 
-  return (
-    <circle
-      cx={props.point.x}
-      cy={props.point.y}
-      r={r}
-      {...extraProps}
-    />
-  )
-})
+    return <circle key={r} cx={props.point.x} cy={props.point.y} r={r} {...extraProps} />
```

Comment:
The real change in this file is that I added a key because this returns an array